test/verify/check-realms

```diff
@@ -257,12 +243,7 @@ class TestRealms(MachineCase):
         b.click("#system-info-domain a")
         b.wait_popup("realms-op")
         b.set_val(self.op_address, "NOPE")
-        if m.image not in ["rhel-8-0-distropkg"]:
-            # changed in PR #11022
-            b.wait_text(".realms-op-address-error", "Domain NOPE could not be contacted")
-            b.wait_not_visible(".realms-op-address-spinner")
-        else:
-            b.wait_attr(".realms-op-address-error", "title", "Domain NOPE could not be contacted")
+        b.wait_attr(".realms-op-address-error", "title", "Domain NOPE could not be contacted")
```

Comment:
You kept the wrong branch here :) (https://209.132.184.41:8493/logs/pull-12735-20190910-070707-23002eae-cockpit-project-cockpit--rhel-8-1-distropkg/log.html#92-2)

Reply:
Argh, lots of `in` vs `not in`.. fixed.